vm/vm.go

```diff
@@ -1128,6 +1141,46 @@ func (vm *VM) loadAcceptedBlocks(ctx context.Context) error {
 	return nil
 }
 
+func (vm *VM) restoreAcceptedQueue(ctx context.Context) error {
+	has, err := vm.HasLastProcessed()
+	if err != nil {
+		return fmt.Errorf("could not determine if have last processed: %w", err)
```

Comment:
if have last processed is a bit hard to read.

```suggestion
		return fmt.Errorf("could not load last processed block: %w", err)
```